jenkins-master/jobs/scripts/workspace/virtualenv/.gitignore

```diff
@@ -0,0 +1,10 @@
+virtualenv.egg-info
```

Comment:
Mirroring this is not ideal. A .zip file is better. Is there a reason it's not just an environment requirement?